pkg/api/customization/project/project_store.go

```diff
@@ -31,11 +39,26 @@ func (s *projectStore) Create(apiContext *types.APIContext, schema *types.Schema
 		return nil, err
 	}
 
+	if err := s.validateResourceQuota(apiContext, schema, data, ""); err != nil {
+		return nil, err
+	}
+
 	values.PutValue(data, annotation, "annotations", roleTemplatesRequired)
 
 	return s.Store.Create(apiContext, schema, data)
 }
 
+func (s *projectStore) Update(apiContext *types.APIContext, schema *types.Schema, data map[string]interface{}, id string) (map[string]interface{}, error) {
+	if err := s.validateResourceQuota(apiContext, schema, data, id); err != nil {
+		return nil, err
+	}
+	if err := s.validateResourceQuota(apiContext, schema, data, ""); err != nil {
```

Comment:
WHy do you call this twice back-to-back with just the id paramter differing?
Seems like the implementation of validateResourceQuota doesnt use the id parameter at all.

Reply:
@cjellick copy/paste error :) will fix